src/app/core/app.component.html

```diff
@@ -49,7 +49,7 @@
 </div>
```

Comment:
When chosing to also remove groups that are empty with a mix of empty and populated groups in the selection, the behavior is a weird:
1. If the group is not empty, we still send an xhr request that fails, I doubt we should send it if we know the group is not empty
2. If a group cannot be deleted it won't be removed from the sub groups list. I think the expected behavior is that it should removed from the subgroups list but not deleted

The 2 points are probably correlated, shouldn't we proceed this way for the "remove groups that are empty" scenario:
:arrow_right: if group is empty, call remove group service ; else call "group remove child"
?

What do you think of the current behavior and the proposed one, do you see something else we could/should do ?

Reply:
Yes, I guess so lot changes need to approve with @smadbe or maybe create new proposal issue for improve this logic

Reply:
Issue created